controllers/twophase/types.go

```diff
@@ -73,13 +80,23 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		targetPhase = v1alpha1.ExperimentPhaseRunning
 	}
 
-	if chaos.IsPaused() {
+	if chaos.GetPause() != "" {
 		targetPhase = v1alpha1.ExperimentPhasePaused
+		// pause for certain duration, only set autoResume once
+		if chaos.GetPause() != "true" && status.Experiment.Phase != v1alpha1.ExperimentPhasePaused {
```

Comment:
Why not add a single annotation to represent the pause time?

Reply:
The reason I use the same annotation is that [as doc recommends](https://chaos-mesh.org/docs/user_guides/run_chaos_experiment#resume-a-chaos-experiment), we set annotation to nil to resume pause. And I think that's the same pattern we could also use here, from duration to nil.

If it would make you or maybe users confused, I could add another annotation like `pauseDuration`, compared to the original `pause`.

Reply:
I think to add another annotation like `pauseDuration` is better.